web-wallet/src/lib/stores/gasStore.js

```diff
@@ -0,0 +1,33 @@
+import { readable } from "svelte/store";
+import { isType } from "lamb";
+
+const gasSettings = {
+	limitLower: Number(parseInt(import.meta.env.VITE_GAS_LIMIT_LOWER, 10)),
+	limitUpper: Number(parseInt(import.meta.env.VITE_GAS_LIMIT_UPPER, 10)),
+	priceLower: Number(parseInt(import.meta.env.VITE_GAS_PRICE_LOWER, 10))
+};
+
+const gasStore = readable(gasSettings);
+const { subscribe } = gasStore;
+
+/**
+ *
+ * @param {Number} price
+ * @param {Number} limit
+ * @returns {Boolean}
+ */
+const areValidSettings = (price, limit) => {
+	let isValidPrice = false;
+	let	isValidLimit = false;
```

Comment:
A small issue with the whitespace here.

Reply:
This is interesting, I do not see it locally.

Reply:
Me neither, I had to remove the space and add it again, and save the file, so to see it.

<img width="1943" alt="Screenshot 2024-02-26 at 15 21 46" src="https://github.com/dusk-network/rusk/assets/34094428/e21a39c8-fe5d-49bd-a5f5-4ca62add3cde">

But in the file it looks the same.

Reply:
I have pushed a fix for it ([reference](https://github.com/dusk-network/rusk/pull/1451/files#diff-12168cae6779670447792b424ae90bb1f5a7140b05287247531b8d7c566ae1d4R20-R22)).